lib/internal/bootstrap_node.js

```diff
@@ -249,6 +250,21 @@
     });
   }
 
+  function setupInspectorAPI() {
+    const addCommandLineAPIMethod = process.inspector.addCommandLineAPIMethod;
+    addCommandLineAPIMethod('require', function require(request) {
```

Comment:
The line you're commenting on is executed synchronously during bootup. From my understanding that should prevent any debug client from connecting (and calling `Runtime.evaluate`) before we registered the API method.